components/TextFields/examples/MDCBaseTextFieldTypicalUseExample.m

```diff
@@ -30,7 +30,10 @@ @interface MDCBaseTextFieldTypicalExampleViewController : UIViewController
 @property(nonatomic, strong) MDCBaseTextField *baseTextField;
```

Comment:
File comment: should we rename this example controller to remove `base`?
While your at it I would add `controller` into the name of the file: MDCTextFieldTypicalExampleViewController

Reply:
I'm going to replace this whole example pretty soon with one I wrote previously so I wasn't going to bother with that, but I can!

Reply:
Renamed example 😃